docs/guides/troubleshooting.md

```diff
@@ -75,15 +72,14 @@ curl -s -H "Authorization: Bearer $PREFECT_API_KEY" "https://api.prefect.cloud/a
 ```
 
 !!! note "Formatting JSON"
-    Python comes with a helpful [tool](https://docs.python.org/3/library/json.html#module-json.tool) for formatting JSON. Append the following to the end of the command above to make the output more readable: ` | python -m json.tool`
-
-Make sure your actor is a member of the workspace you are working in. Within a workspace, an actor has a [role](/cloud/users/roles/) which grants them certain permissions. Insufficient permissions will result in an error. For example, starting an agent or worker with the __Viewer__ role, will result in errors.
+    Python comes with a helpful [tool](https://docs.python.org/3/library/json.html#module-json.tool) for formatting JSON. Append the following to the end of the command above to make the output more readable: `| python -m json.tool`
 
+Make sure your actor is a member of the workspace you are working in. Within a workspace, an actor has a [role](/cloud/users/roles/) which grants them certain permissions. Insufficient permissions will result in an error. For example, starting an agent or worker with the **Viewer** role, will result in errors.
 
 ## Execution
 
-Prefect flows can be executed locally by the user, or remotely by an agent or worker. Local execution generally means that you - the user - run your flow directly with a command like `python flow.py`. Remote execution generally means that an agent or worker runs your flow via a [deployment](/concepts/deployments/), optionally on different infrastructure.
+Prefect flows can be executed locally by the user, or remotely by a worker or agent. Local execution generally means that you - the user - run your flow directly with a command like `python flow.py`. Remote execution generally means that a worker runs your flow via a [deployment](/concepts/deployments/), optionally on different infrastructure.
 
-With remote execution, the creation of your flow run happens separately from its execution. Flow runs are assigned to a work pool and a work queue. In order for flow runs to execute, an agent or worker must be subscribed to the work pool and work queue, otherwise the flow runs will go from `Scheduled` to `Late`. Ensure that your work pool and work queue have a subscribed agent or worker.
+With remote execution, the creation of your flow run happens separately from its execution. Flow runs are assigned to a work pool and a work queue. For flow runs to execute, an worker must be subscribed to the work pool and work queue, otherwise the flow runs will go from `Scheduled` to `Late`. Ensure that your work pool and work queue have a subscribed worker.
```

Comment:
```suggestion
With remote execution, the creation of your flow run happens separately from its execution. Flow runs are assigned to a work pool and a work queue. For flow runs to execute, a worker must be subscribed to the work pool and work queue, otherwise the flow runs will go from `Scheduled` to `Late`. Ensure that your work pool and work queue have a subscribed worker.
```